docs/api_reference.rst

```diff
@@ -23,13 +23,13 @@ Modules
 
 .. toctree::
    :maxdepth: 1
-
+   
```

Comment:
Please do not add blank spaces (if you use the editorconfig plugin it will remove trailing whitespaces automatically ;))